org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/alloy4/TableView.java

```diff
@@ -255,6 +269,14 @@ private static SimTuple toSimTuple(A4Tuple a4t) {
         return SimTuple.make(atoms);
     }
 
+    private static List<SimTuple> toArrayList(TupleSet tupleSet) {
+        List<SimTuple> l = new ArrayList<>();
```

Comment:
pass the size of `tupleSet` to the constructor of `ArrayList` to initialize its capacity